packages/std/src/math/int128.rs

```diff
@@ -287,6 +289,16 @@ impl From<i8> for Int128 {
     }
 }
 
+impl TryFrom<Int256> for Int128 {
+    type Error = ConversionOverflowError;
+
+    fn try_from(value: Int256) -> Result<Self, Self::Error> {
+        shrink_be_int(value.to_be_bytes())
+            .ok_or_else(|| ConversionOverflowError::new("Int256", "Int128", value))
+            .map(Self::from_be_bytes)
+    }
+}
```

Comment:
Can you add `impl TryFrom<Int512> for Int128 {` too?

Reply:
Yes, I'll add those too